Add Go snippets for file attachments (#204)
xataio/mdx-docs: 030-Concepts/075-file-attachments.mdx

![Entire columns can be made public by default](images/column-access-public.png)

<Alert status="warning">

>   **Public object caching** When a public object gets cached, changing permissions will not invalidate the cached entry.

Reviewer: ```suggestion
  **Public object caching** Changing the permissions of a cached public object does not remove it from the cache; the cached version remains unaffected.
```

>   The file is still accessible through old URLs until the cache entry expires in 2h. Xata advises greatest caution when

Reviewer: ```suggestion
  The file remains accessible through the previously existing URLs until the cache expires after 2 hours. It is highly recommended to exercise caution when
```

>   configuring public access, because in practice there is a delay in changing the permissions back from public to

Reviewer: ```suggestion
  configuring public access. There is a significant delay in reverting permissions from public back to 
```
